docs/products/kafka/kafka-connect/getting-started.rst

```diff
@@ -32,10 +30,9 @@ Create a new Aiven for Apache Kafka Connect dedicated service:
 
 6. Select a service plan. This defines how many servers and what kind of memory, CPU, and disk resources are allocated to your service.
 
-7. Click **Create Service** under the summary on the right side of the console. 
-
+7. Click **Create and enable** under the summary in the console. 
 
-You can see the service status in the **Service overview** page , wait till it changes from *REBUILDING* to *RUNNING* to be able to use its full functionality.
+At the top of the screen, you will notice the Apache Kafka Connect integration. Clicking on the service name will take you to the **Service overview** page, where you can monitor the service status. It's important to wait until the service status changes from *REBUILDING* to *RUNNING* on this page before using its full functionality.
```

Comment:
minor thing - `its` in `its full functionality`. It's a little bit unclear what it refers to. It's replace with `full service functionality` or `the full functionality`

Reply:
> minor thing - `its` in `its full functionality`. It's a little bit unclear what it refers to. It's replace with `full service functionality` or `the full functionality`

Kafka Connect distributed Apache Kafka® integration component - removed 'it's full functionality" so the line is "It's important to wait until the service status changes from *REBUILDING* to *RUNNING* on this page before using it." - meaning before using the service.